### ISSUE TITLE
Add a scrollbar

### ISSUE THREAD

**Issue by emanur** [Wikiki/bulma-quickview#15]
My content for the Quickview exceeds the height of the viewport but there is no vertical scrollbar. 

![image](https://user-images.githubusercontent.com/49459452/55838327-00137d80-5b1c-11e9-99d4-d7de88e44411.png)

The browser scrolls the whole page but behind the quickview. It's almost like the Quickview is behaving like a modal.

How can I add a scrollbar to Quickview? I appreciate this will result in double scrollbars but that's okay for my use case. I took a look at #3 but it didn't really help me as such and is in any case now marked closed.

Thanks in advance :)

**Comment by dryhurst** [comment]
@emanur the `quickview` in my app shows a scroll bar by default when the height of the `quickview` exceeds that of the browser. make sure you are using the most recent version.

**Comment by Wikiki** [comment]
closed due to inactivity

**Comment by francesco-loreti** [comment]
Hi, I had the same problem and I had to countless attempts, I understood the problem and luckily I got around it but I could not solve it.
Basically if a simple list of elements is inserted in the quickview, the scrollbar appears correctly. 
Different, as in the case indicated above, if you insert a form tag inside the quickview. 
In this case the scrollbar does not appear and I have not found a way to make it appear. 
The only alternative therefore was to get around the problem by putting the form tag inside the `div.quickview-body` . 
At this point the scrollbar finally appears. The bad thing is that if the quickview is on the right and opens to the left, the scrollbar overlaps the page scrollbar and it becomes difficult to manage. 
It would be optimal to insert a css class to allow the scrollbar to be displayed on the opposite side (on the left).
